zerver/actions/custom_profile_fields.py

```diff
@@ -101,14 +101,17 @@ def remove_custom_profile_field_value_if_required(
 def try_update_realm_custom_profile_field(
     realm: Realm,
     field: CustomProfileField,
-    name: str,
-    hint: str = "",
+    name: Optional[str] = None,
+    hint: Optional[str] = None,
     field_data: Optional[ProfileFieldData] = None,
     display_in_profile_summary: bool = False,
     required: bool = False,
 ) -> None:
-    field.name = name
-    field.hint = hint
+    if name:
```

Comment:
Name cannot be empty string, so there is no need for checking whether it is None or not

Reply:
It's still better code to write `if name is not None`; always good to be explicit about what falsey values are expected.

Reply:
Fixed.